gulpfile.js

```diff
@@ -54,6 +54,11 @@ gulp.task('underscore', function() {
     gulp.src('bower_components/underscore/**/*')
         .pipe(gulp.dest('./build/public/lib/underscore'));
 });
+// gulp.task('materialize', function() {
+//   gulp
+//     .src('bower_components/materialize/**/*')
+//     .pipe(gulp.dest('./build/public/lib/materialize'));
+// });
```

Comment:
Any specific reason why this is commented out?

Reply:
It is useless, I will remove it